.pylintrc-packaged-files

```diff
@@ -1,10 +1,12 @@
 [MESSAGES CONTROL]
-disable=missing-docstring,
-        duplicate-code, ### Temporarily disabled until sample files begin to look different
+disable=duplicate-code, ### Temporarily disabled until sample files begin to look different
         bad-continuation, ### Disabled because in conflict with black
 
 [FORMAT]
 max-line-length=120
 
 # Allow long lines if they contains long strings
 ignore-long-lines="[^"]{80,}"
+# While camera.capture() can return both Frame and Frame2D, pylint only thinks it can return
+# Frame. Therefore it will complain with "no-member" on methods that only exist on Frame2D.
```

Comment:
It's basically a weakness in pylint. It doesn't understand that two different code paths return two different types.

A "fix" would be to go back to having separate "capture()" and "capture2d()", but that's not really in the API 2.0 spirit. What do you think?

Reply:
On the other hand, having "capture()" and "capture2d()" feels more Pythonic because effectively overloading based on input type is kind of uncommon.

Reply:
I do not have an opinion on overload or not right not. But we should generally not change the API based on a limitation (we do not agree on) in a linter. If you think this the best API, then keep it. But, improve the comment.

Reply:
I believe this is the best API choice, since we want the naming to be analogous to the C++ API (and the C++ API doesn't have a capture2d function). I improved the comment.